flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
but yet you try to unpack an aarch64 binary as well. Just remove this.

Reply:
we had a problem with flatpak on ARM so I added this temporarily to avoid users downloading it on that architecture

Reply:
then don't put it in the manifest, it's misleading. and disappoiting.